chaoshub/cli.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+import io
+import json
+from logzero import logger
+
+import click
+
+__all__ = ["login"]
+
+from chaoshub.publish import publish_to_hub
+from chaoslib.settings import load_settings, save_settings
```

Comment:
they should go before the `import click` statement as this is a third-part lib

Reply:
Done.